composer/loggers/mosaicml_logger.py

```diff
@@ -113,6 +113,8 @@ def epoch_end(self, state: State, logger: Logger) -> None:
         self._flush_metadata()
 
     def fit_end(self, state: State, logger: Logger) -> None:
+        # Log model training finished time for run events
+        self._log_metadata({'train_finished_time': time.time()})
```

Comment:
Should this subtract off start time? or do u want absolute

Reply:
I am reporting the timestamp for when training finishes so we want absolute!